docs/index.rst

```diff
@@ -1,22 +1,42 @@
-Welcome to POCS's documentation!
-================================
+PANOPTES Observatory Control System - 	POCS
+=====
 
-![PANOPTES prototype unit on Mauna Loa](http://projectpanoptes.org/v1/wp-content/uploads/2013/02/System_view21.png)
+`PANOPTES <http://projectpanoptes.org>`_ is an open source citizen science project that is designed to find exoplanets with digital cameras. The goal of PANOPTES is to establish a global network of of robotic cameras run by amateur astronomers and schools in order to monitor, as continuously as possible, a very large number of stars. For more general information about the project, including the science case and resources for interested individuals, see the `project website <http://projectpanoptes.org/>`_.
```

Comment:
If the file format allows, I'd certainly prefer shorter lines.

Reply:
Done.